tools/Environments/DevHome.Environments/ViewModels/ComputeSystemViewModel.cs

```diff
@@ -308,10 +314,12 @@ private void LogFailure(ComputeSystemOperationResult? operationResult)
         else
         {
             _log.Error(operationResult.Result.ExtendedError, $"Launch operation failed for {ComputeSystem} error: {operationResult.Result.DiagnosticText}");
+            diagnosticText = operationResult.Result.DiagnosticText;
         }
 
         // Show the error notification to tell the user the operation failed
         OnErrorReceived(errorMessage);
+        return (errorMessage, diagnosticText);

```

Comment:
If operations is null then errorMessage and diagnosticText are the same.  Is that okay?

Reply:
thanks I'll update the messaging for the null case